test/pkg/test/framework.go

```diff
@@ -277,8 +266,15 @@ func (env *Framework) GetTargets(ctx context.Context, targetGroup *vpclattice.Ta
 	var found []*vpclattice.TargetSummary
 	Eventually(func(g Gomega) {
 		log.Println("Trying to retrieve registered targets for targetGroup", targetGroup)
+		log.Println("deployment.Spec.Selector.MatchLabels:", deployment.Spec.Selector.MatchLabels)
 		podList := &v1.PodList{}
-		g.Expect(env.List(ctx, podList, client.MatchingLabels(deployment.Spec.Selector.MatchLabels))).To(Succeed())
+		expectedMatchingLabels := make(map[string]string, len(deployment.Spec.Selector.MatchLabels))
```

Comment:
Is it possible these are the one causing my 2nd test stucked waiting for target?